pkg/reconciler/dependencybuild/buildrecipeyaml.go

```diff
@@ -362,87 +359,145 @@ func createPipelineSpec(log logr.Logger, tool string, commitTime int64, jbsConfi
 	runAfterBuild = append(runAfter, BuildTaskName)
 
 	ps := &tektonpipeline.PipelineSpec{
-		Workspaces: []tektonpipeline.PipelineWorkspaceDeclaration{{Name: WorkspaceBuildSettings}, {Name: WorkspaceSource}, {Name: WorkspaceTls}},
+		Workspaces: []tektonpipeline.PipelineWorkspaceDeclaration{{Name: WorkspaceSource}, {Name: WorkspaceTls}},
 	}
 
 	if preBuildImageRequired {
-		buildSetup := tektonpipeline.TaskSpec{
-			Workspaces: []tektonpipeline.WorkspaceDeclaration{{Name: WorkspaceBuildSettings}, {Name: WorkspaceSource, MountPath: WorkspaceMount}, {Name: WorkspaceTls}},
-			Params:     pipelineParams,
-			Results: []tektonpipeline.TaskResult{
-				{Name: PipelineResultPreBuildImageDigest, Type: tektonpipeline.ResultsTypeString},
-				{Name: PipelineResultGitArchive, Type: tektonpipeline.ResultsTypeString},
+		resolver := tektonpipeline.ResolverRef{
+			// We can use either a http or git resolver. Using http as avoids cloning an entire repository.
+			Resolver: "http",
+			Params: []tektonpipeline.Param{
+				{
+					Name: "url",
+					Value: tektonpipeline.ParamValue{
+						Type:      tektonpipeline.ParamTypeString,
+						StringVal: "https://raw.githubusercontent.com/rnc/jvm-build-service/KJB11/deploy/tasks/pre-build.yaml",
```

Comment:
As per prior PRs will need a follow-up PR to correct the branch/repo.

Reply:
+1